staging/src/k8s.io/apiserver/pkg/storage/cacher/delegator.go

```diff
@@ -33,18 +41,41 @@ import (
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/component-base/tracing"
 	"k8s.io/klog/v2"
+	"k8s.io/utils/clock"
+)
+
+var (
+	ConsistencyCheckPeriod = 5 * time.Minute
```

Comment:
Given that we want to enabled it in CI, I think we should be doing that more frequently than every 5m.

Reply:
5 minutes were proposed in KEP as compatible time to compaction. Value seemed right as magnitude higher than LIST latency SLO (30 seconds), and timeout (1 minute) . Don't have strong feelings about it. 

For CI I expect most e2e tests take much longer than 5 minutes. For integration tests I lower the value in test.

Reply:
OK - I guess we can start with that at least.

Please add a comment explaining that though.

Reply:
Done